examples/modis-061/modis-14A1-061/MYD14A1.A2022025.h01v07.061.2022035001141/MYD14A1.A2022025.h01v07.061.2022035001141.json

```diff
@@ -535,6 +389,48 @@
           "summary": "Fire (high confidence, land or water)"
         }
       ],
+      "classification:classes": [
```

Comment:
is it intentional to include both file:value and classification:classes?

Reply:
Yes, for now Microsoft wants us to include both so they can demo the classification extension while keeping their existing file:values functionality.